tests/test_controlnet_maisi.py

```diff
@@ -103,16 +101,17 @@
 TEST_CASES_ERROR = [
     [
         {"spatial_dims": 2, "in_channels": 1, "with_conditioning": True, "cross_attention_dim": None},
-        "ControlNet expects dimension of the cross-attention conditioning "
-        "(cross_attention_dim) when using with_conditioning.",
+        "DiffusionModelUNet expects dimension of the cross-attention conditioning (cross_attention_dim) "
```

Comment:
Hi @guopengf, could you please help modify the error message in https://github.com/Project-MONAI/MONAI/blob/62430315cd176478d06dd197b7a6dfdd6cd90c44/monai/networks/nets/controlnet.py#L181?

Look like it's wrong and misleading.

Reply:
Yeah, change it to ControlNet